frontend/src/App.vue

```diff
@@ -183,6 +192,16 @@ export default {
     this.checkStorage();
 
     Events.$on('setting:rewards', () => this.checkStorage());
+
+    document.body.addEventListener('click', (e) => {
+      if(e.target.nodeName==='BUTTON') {
+        window.gtag('event', 'button_clicked', {clickInfo: e.target.getAttribute('tagname')});
+      }
+
+      if (e.target.id === 'gtag-link-others') { // capture clickable elements
```

Comment:
ids are unique, you can't have multiple of the same one in the same page - this makes viewlinks invalid. you should use a class for this. past that, it looks great!